.gitignore

```diff
@@ -41,6 +41,7 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+CHANGELOG.md
```

Comment:
Does this mean that there is no changelog file that is part of the git repo?
So the changelog is always dynamically generated?

Reply:
Unfortunately there is no way to keep the changelog in the repo and update it automatically. Because each change in the changelog will always need a commit and we can't commit automatically to base branch. The changelog is anyway not up-to-date in the base branch. 
We decided to generate it automatically and add it to the release assets and to the build before publishing to npm.

Reply:
So it's always accessible in the release assets as a file?

Reply:
Yes, exactly

Reply:
Ah, then it's totally fine.